examples/application_commands/slash_command.py

```diff
@@ -0,0 +1,13 @@
+import nextcord
+from nextcord.ext import commands
+
+client = nextcord.Client()
+
+
+@client.slash_command(guild_ids=[...]) # limits guilds with this command.
+async def ping(interaction):  # Passing Through interaction And The Name.
```

Comment:
```suggestion
async def ping(interaction: nextcord.Interaction):  # Passing Through interaction And The Name.
```
Typehinting should be added here for consistency.